build/Makefile.conf

```diff
@@ -1,6 +1,6 @@
-F90      = ifort
-F90FLAGS = -O3
+#F90      = ifort
+F90FLAGS = -O3 -ffree-line-length-none
```

Comment:
Would it make sense to run with more complete debugging flags, e.g. -ffree-line-length-none fbounds-check -finit-real=nan ? What about -std=f2008 ?

Reply:
I've added the first two flags, but left out -std=f2008 for the moment. It throws lots of 'Nonconforming tab character' warnings. I propose to open a open an issue to delete the tabs first, and then introduce that flag.